canonical-web-and-design/maas-ui
# refactor(shared): use the Navigation component from react-components

## Done

- Remove the Navigation component and use the one provided by react-components.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Run the full client.
- Check that the main navigation works for both the React and AngularJS clients.

## Fixes

Fixes: #3841.
